backend/src/handlers/import_operations.py

```
@@ -0,0 +1,303 @@
+import json
+import logging
+import uuid
```

Ruff (0.12.2)

`uuid` imported but unused

Remove unused import: `uuid`

(F401)